clusterctl/examples/google/generate-yaml.sh

```diff
@@ -37,7 +37,7 @@ MACHINE_CONTROLLER_SSH_PUBLIC=
 MACHINE_CONTROLLER_SSH_PRIVATE_FILE=${OUTPUT_DIR}/machine-controller-key
 MACHINE_CONTROLLER_SSH_PRIVATE=
 MACHINE_CONTROLLER_SSH_USER_PLAIN=clusterapi
-MACHINE_CONTROLLER_SSH_USER=$(echo -n "$MACHINE_CONTROLLER_SSH_USER_PLAIN" | base64 -w0)
+MACHINE_CONTROLLER_SSH_USER=$(echo -n "$MACHINE_CONTROLLER_SSH_USER_PLAIN" | base64 | tr -d '\r\n')
```

Comment:
nit: can you document the reason for tr -d similar to what you did in the other PR.